glue_jupyter/bqplot/common/tools.py

```diff
@@ -25,6 +25,24 @@ def deactivate(self):
         self.viewer._mouse_interact.next = None
 
 
+class BqplotSelectionTool(InteractCheckableTool):
+    """Disables all other tools when activated."""
+
+    def activate(self):
+        with self.viewer._output_widget:
+            self.interact.selected_x = None
+            self.interact.selected_y = None
+
+        for viewer in self.viewer.session.application.viewers:
+            if viewer is not self.viewer:
+                viewer.toolbar.active_tool = None
+
+        # TODO: Tom asked me to add this, but adding this fails the test.
+        # self.viewer.session.edit_subset_mode.edit_subset = []
```

Comment:
@astrofrog -- What does this line do anyway? The deactivation seems to work fine without it. Enabling this line breaks a test in the CI here.

Reply:
This was the line that should make it change back to 'Create New' - basically edit_subset is the list of subsets currently being modified, and if it's empty it should then become 'Create New'

Reply:
Are you expecting the CI to fail with this change? Any idea how to fix?